include/ocpp/v201/ocpp_types.hpp

```diff
@@ -383,10 +1024,10 @@ std::ostream& operator<<(std::ostream& os, const GetVariableResult& k);
 
 struct SignedMeterValue {
     CiString<2500> signedMeterData;
-    CiString<50> signingMethod;
     CiString<50> encodingMethod;
-    CiString<2500> publicKey;
     std::optional<CustomData> customData;
+    std::optional<CiString<50>> signingMethod;
+    std::optional<CiString<2500>> publicKey;
```

Comment:
TODO: find out implications for backwards compatibility with 2.0.1
These two fields were required in 2.0.1 and are now optional in 2.1